src/js/expandCollapseButton.js

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2021 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
+*/
+
+"use strict";
+
+fluid.registerNamespace("fluid.expandCollapseButton");
+
+// The expand-collapse button component. This button is only available on the mobile view.
+// To use this component, add a model listener for the path "expanded", for example:
+// expandCollapse: {
+//     path: "expanded",
+//     func: "{that}.toggleExpandedState",
+//     args: ["{change}.value", ["{that}.dom.title", "{that}.dom.cities"]]
+// }
+// Note that the invoker "toggleExpandedState" requires 2 arguments:
+// 1. The changed model value;
+// 2. An array of DOM elements to show/hide.
+fluid.defaults("fluid.expandCollapseButton", {
+    gradeNames: "fluid.viewComponent",
+    selectors: {
+        expandCollapseButton: ".fl-mapviz-expand-collapse-button"
+    },
+    model: {
+        expanded: true
+    },
+    invokers: {
+        toggleExpandedState: {
```

Comment:
I've reworked this component to make it fully integral and removing the need for invokers and overrides like this in my branch https://github.com/cindyli/covid-data-monitor/pull/1